### ISSUE TITLE
gnome-control-center (Ubuntu Settings App) can't handle buttons on left side

### ISSUE THREAD

**Issue by nicolas-oliveira** [vinceliuice/WhiteSur-gtk-theme#1021]
I know, I know, this is an app problem, but I really need to open this issue first and look some fix alternatives:
![image](https://github.com/vinceliuice/WhiteSur-gtk-theme/assets/45501748/89f9b58a-a907-4fe6-814f-43c2a189997b)

This glitch really concern me, because I search some solution since last year, but with no solutions
this glitch is very trivial and also visual, but it's annoying nonetheless. Especially if someone is applying this theme and trying to look and feel like "mac"...
And again I know it's an application problem, as I mentioned here:  https://github.com/vinceliuice/WhiteSur-gtk-theme/issues/556

But I've found this bug in gitlab and launchpad but for all of them it's an low-priority  issue: [gitlab.gnome #1217](https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/1217) , [gitlab.gnome #1037](https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/1037) and [launchpad #1873638](https://bugs.launchpad.net/ubuntu/+source/gnome-control-center/+bug/1873638)

But I think even if this is an application problem, is there any way around it?
People who use this theme and have come across this glitch are sure to be interested in a solution...

Anyone have suggestions?

<details>
<summary>Some infos</summary>

```
Ubuntu 22.04.3 LTS 
GNOME 42.9

CPU: 12-core (4-mt/8-st) 12th Gen Intel Core i5-12500H (-MST AMCP-)
speed/min/max: 2938/400/4500:3300 MHz Kernel: 6.2.0-26-generic x86_64 Up: 23m
Mem: 4302.5/7617.0 MiB (56.5%) Storage: 238.47 GiB (70.7% used) Procs: 424
Shell: fish inxi: 3.3.13
Graphics:
  Device-1: Intel Alder Lake-P Integrated Graphics driver: i915 v: kernel
  Device-2: NVIDIA GA107BM [GeForce RTX 3050 Mobile] driver: nvidia
    v: 550.40.07
  Device-3: Microdia Integrated_Webcam_HD type: USB driver: uvcvideo
  Display: x11 server: X.Org v: 1.21.1.4 driver: X:
    loaded: modesetting,nvidia unloaded: fbdev,nouveau,vesa gpu: i915
    resolution: 1: 1920x1080~60Hz 2: 1600x900~120Hz
  OpenGL: renderer: NVIDIA GeForce RTX 3050 Laptop GPU/PCIe/SSE2
    v: 4.6.0 NVIDIA 550.40.07

```
</details>

**Comment by kernelb00t** [comment]
Hello, does this issue still occurs? Please test with the latest LTS (22.04.4) or latest stable (23.10).

**Comment by kernelb00t** [comment]
Hey @nicolas-oliveira :arrow_up:

**Comment by nicolas-oliveira** [comment]
Yes still occours. 
![image](https://github.com/user-attachments/assets/80d57a61-23c5-4472-a2df-271f24345c9f)

The gnome-tweaks doesn't have this issue:
![image](https://github.com/user-attachments/assets/7c6dcd16-23c8-4a7c-bf36-92903b13bc46)

**Comment by vinceliuice** [comment]
Not theme issue

**Comment by kernelb00t** [comment]
I'll get a Ubuntu beta VM and check that this bug still occurs. If it's easily fixable I'll try. I'm not very familiar with GTK, but this will be the occasion to learn it.

**Comment by nicolas-oliveira** [comment]
Yes @vinceliuice as I said before this is an app issue, I even listed the issues in others places. I only opened this issue because maybe some users of the theme can face this problem...
